lib/biomarker.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Use the same title for biomarker search page and biomarker object page for that biomarker
+ */
+
+export function getTitle(biomarker) {
+  const titleElements = [biomarker.name, biomarker.quantification].filter(
+    Boolean
+  );
```

Comment:
It looks like both `name` and `quantification` are both required in the schema, so you technically don’t need the Boolean filter. I could see an argument to keep the filter in case either of them get changed to non-required, though if they both become non-required, we’d need a fallback in case neither exists.